src/functions/template-tags/general.php

```diff
@@ -1344,8 +1322,24 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 		 */
 		$excerpt_more = apply_filters( 'excerpt_more', ' [&hellip;]' );
 
-		// Now we actually trim it
-		$excerpt = wp_trim_words( $excerpt, $excerpt_length, $excerpt_more );
+		// Trim the excerpt. The following is just wp_trim_words() without its use of wp_strip_all_tags().
+		if ( strpos( _x( 'words', 'Word count type. Do not translate!', 'the-events-calendar' ), 'characters' ) === 0 && preg_match( '/^utf\-?8$/i', get_option( 'blog_charset' ) ) ) {
+			$excerpt = trim( preg_replace( "/[\n\r\t ]+/", ' ', $excerpt ), ' ' );
+			preg_match_all( '/./u', $text, $words_array );
+			$words_array = array_slice( $words_array[0], 0, $excerpt_length + 1 );
+			$sep = '';
+		} else {
+			$words_array = preg_split( "/[\n\r\t ]+/", $excerpt, $excerpt_length + 1, PREG_SPLIT_NO_EMPTY );
+			$sep = ' ';
+		}
+
+		if ( count( $words_array ) > $excerpt_length ) {
+			array_pop( $words_array );
+			$excerpt = implode( $sep, $words_array );
+			$excerpt = $excerpt . $excerpt_more;
+		} else {
+			$excerpt = implode( $sep, $words_array );
+		}
 
 		return wpautop( $excerpt );
```

Comment:
Should we provide a filter here before returning the excerpt, basically echoing what `wp_trim_words()` does?